docs/source/chatops/aliases.rst

```diff
@@ -369,6 +369,59 @@ dynamically:
 Support for other chat providers is coming soon, and of course you are always welcome to
 contribute! See the example below for hacking on ``extra``.
 
+Mattermost and Rocketchat also support the Slack attachments API. However, you will need
+to use the ``mattermost`` and ``rocketchat`` keys to ``extra``:
```

Comment:
"you will need to add .... to extra"?

Reply:
Fixed my grammar to be more clear about what I'm pointing out.